docs/source/reference/numpysupported.rst

```diff
@@ -133,6 +133,80 @@ The following constructors are supported, only with a numeric input:
 * :class:`numpy.uintp`
 
 
+Modules
+=======
+
+``random``
+----------
+
+Numba supports top-level functions from the
+`numpy.random <http://docs.scipy.org/doc/numpy/reference/routines.random.html>`_
+module, but does not allow you to create individual RandomState instances.
+The same algorithms are used as for :ref:`the standard
+random module <pysupported-random>` (and therefore the same notes apply),
+but with an independent internal state: seeding or drawning numbers from
+one generator won't affect the other.
+
+The following functions are supported, but only with scalar output: you can't
+pass a *size* argument.
+
```

Comment:
Do we need the same `fork()` warning for the Numpy RNG as we have for the Python RNG?